float64 roll_deg
float64 pitch_deg
float64 yaw_deg
uint32 x
uint32 y
uint32 z